hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/FSQueueConverter.java

```diff
@@ -309,6 +314,18 @@ private void checkMaxChildCapacitySetting(FSQueue queue) {
     }
   }
 
+  private boolean checkAutoQueueCreationV2Disabled(String queueName) {
+    return !capacitySchedulerConfig.getBoolean(
+        PREFIX + queueName + DOT + AUTO_QUEUE_CREATION_V2_ENABLED,
+        DEFAULT_AUTO_QUEUE_CREATION_ENABLED);
+  }
+
+  private boolean checkAutoCreateChildQueueDisabled(String queueName) {
+    return !capacitySchedulerConfig.getBoolean(
+        PREFIX + queueName + DOT + AUTO_CREATE_CHILD_QUEUE_ENABLED,
+        DEFAULT_AUTO_CREATE_CHILD_QUEUE_ENABLED);
+  }
```

Comment:
These methods already exist in CapacitySchedulerConfiguration, mentioned the method names in my previous comments. Please use those.

Reply:
Ohh, the tricky thing is that capacitySchedulerConfig is in reality a Configuration object, and not a CapacitySchedulerConfiguration object. This is misleading (and it means that every convenience getter will be reimplemented here), so I've created a follow-up: https://issues.apache.org/jira/browse/YARN-11416. Resolving this conversation for this patch.